### PR TITLE
Update test imports

### PR DESCRIPTION
**Pull request by rebeccaalpert**
Tests seem fine with cleaner imports.

### CODE REVIEW

File: frontend/__tests__/components/catalog.spec.tsx [hunk 1]
```diff
@@ -2,9 +2,9 @@ import * as React from 'react';
 import * as _ from 'lodash-es';
 import { mount, ReactWrapper } from 'enzyme';
 
-import { CatalogTile } from '@patternfly/react-catalog-view-extension/dist/js/components/CatalogTile';
-import { VerticalTabsTab } from '@patternfly/react-catalog-view-extension/dist/js/components/VerticalTabs';
-import { FilterSidePanelCategoryItem } from '@patternfly/react-catalog-view-extension/dist/js/components/FilterSidePanel/FilterSidePanelCategoryItem';
+import { CatalogTile } from '@patternfly/react-catalog-view-extension';
```
Comment (jcaianirh) [inline frontend/__tests__/components/catalog.spec.tsx]: Ok, then lets do this:
`import { 
CatalogTile, 
VerticalTabsTab, 
FilterSidePanelCategoryItem 
} from '@patternfly/react-catalog-view-extension';`

**Review by jcaianirh — CHANGES_REQUESTED** [review]

File: frontend/__tests__/components/catalog.spec.tsx (hunk 1 above)
Comment (rebeccaalpert) [reply frontend/__tests__/components/catalog.spec.tsx]: All set.

**Review by jcaianirh — APPROVED** [review]
lgtm, note prettier may require them to be broken up into one per line

**Review by jcaianirh — APPROVED** [review]
lgtm

**Pull request merged**